.github/workflows/cml-report.yml

```diff
@@ -0,0 +1,100 @@
+name: CML Report
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  sim-summary:
+    if:  ${{ github.event.label.name == 'run-simulation' }}
+    name: CML Simulation
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+    
+    - name: Build API
+      run: |
+        python local/rest_api_gcbm/cml_report/process_input.py
+        cd local/rest_api_gcbm
+        docker build -f Dockerfile.cml --build-arg BUILD_TYPE=RELEASE --build-arg NUM_CPU=4 -t gcbm-api .

```

Comment:
Let's replace this with `docker pull ghcr.io/moja-global/rest_api_gcbm:master` and save ourselves the build step.